.gitignore

```diff
@@ -11,3 +11,4 @@ PHP_BrowscapINI
 ergebnis.txt
 tests/data/*
 !tests/data/.gitkeep
+.php_cs.cache
```

Comment:
It was created on my machine. The cache can exist on your local machine if gitignored, but I'd prefer not to have it committed to the repo tbh.